src/everest_models/jobs/fm_well_trajectory/cli.py

```diff
@@ -36,11 +36,23 @@ def main_entry_point(args=None):
             guide_points,
         )
 
+    # resinsight
     if options.config.connections:
         if (
             eclipse_model := options.eclipse_model or options.config.eclipse_model
         ) is None:
-            args_parser.error("missing eclipse model")
+            args_parser.error("Missing eclipse model")
+
```

Comment:
This seems to test that `eclipse_model` is specified in the config. Do we need to test `f"{eclipse_model}.EGRID").exists()`? Or is that already done somewhere?

If so, does it need a test?

Reply:
I could make a new issue for this. Is it the case if a model is specified there also need to be a EGRID file?

Reply:
I am not sure, I will forward the question to somebody who should know.